docs/source/usage/config.rst

```diff
@@ -117,6 +117,23 @@ These are used if ``model == mtslstm``.
    current input timescale. In both cases, ``transfer_mtslstm_states``
    can be used to configure hidden and cell state transfer.
 
+Transformer settings
+~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~~
```
just to avoid annoying warnings ("Title underline too short") in the docs building process